src/FSharpPlus/Control/Alternative.fs

```diff
@@ -4,6 +4,7 @@ namespace FSharpPlus.Control
 
 open System.Runtime.InteropServices
 open FSharpPlus
+open FSharpPlus.Data
```

Comment:
What about using a type alias `type NonEmptySeq<'t> = FSharpPlus.Data.NonEmptySeq<'t>` instead of opening the whole namespace? Just my 2 cents, feel free to leave it as it is.

Reply:
I think `open FSharpPlus` should not add NonEmptySeq to the scope, so I'll keep this as is.